spacy_llm/tasks/builtin_task.py

```diff
@@ -312,6 +312,14 @@ def _extract_labels_from_example(self, example: Example) -> List[str]:
     def labels(self) -> Tuple[str, ...]:
         return tuple(self._label_dict.values())
 
+    def add_label(self, label: str) -> int:
+        if not isinstance(label, str):
+            raise ValueError(Errors.E187)
+        if label in self.labels:
+            return 0
+        self._label_dict[self._normalizer(label)] = label
+        return 1
```

Comment:
Why do we return 1 here?

Reply:
To be in line with the `TrainablePipe.add_pipe` API: https://spacy.io/api/pipe#add_label

> RETURNS 0 if the label is already present, otherwise 1.